src/main/java/io/reactiverse/pgclient/PgConnection.java

```diff
@@ -83,6 +83,29 @@
   @Fluent
   PgConnection notificationHandler(Handler<PgNotification> handler);
 
+  /**
+   * @return The process ID of the target backend
+   */
+  int getProcessId();
+
+  /**
+   * @return The secret key for the target backend
+   */
+  int getSecretKey();
+
+  /**
+   * Open a new connection to the server and send a CancelRequest message
```

Comment:
send a request cancellation message. No need to ref the actual protocol details here.